commit 7ff4dcb63ce66a112e169fd91bdcaf6b082cd160 (refs/changes/58/4858/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-21T16:29:52+02:00 (1 year, 1 month ago)
    
    TRITON-785 Add requirements.brand and requirements.bootrom to AdminUI's images view

diff --git a/www/js/tpl/image.hbs b/www/js/tpl/image.hbs
index e5bb027f..449fa510 100644
--- a/www/js/tpl/image.hbs
+++ b/www/js/tpl/image.hbs
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 <div class="page-header">
@@ -240,6 +240,20 @@
         </span>
       </li>
       {{/if}}
+
+      {{#if requirements.bootrom}}
+      <li>
+        <strong>BootROM</strong>
+        <span class="value">{{requirements.bootrom}}</span>
+      </li>
+      {{/if}}
+
+      {{#if requirements.brand}}
+      <li>
+        <strong>Brand</strong>
+        <span class="value">{{requirements.brand}}</span>
+      </li>
+      {{/if}}
     </ul>
     </section>
 
